src/main/core/Entity/Log/AbstractLog.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\CoreBundle\Entity\Log;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\MappedSuperclass
+ */
+class AbstractLog
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
```

Comment:
You can use a trait for that : `Claroline\AppBundle\Entity\Identifier\Id`.

Reply:
Done, thx :)